### DISCUSSION TITLE
Error Executing an Input File (Tutorial 01)

### DISCUSSION THREAD

**Discussion by rl3fz** [idaholab/moose#18011] [Q&A Getting Started] [answered]
Hello everyone, I'm on Step 2 of the tutorial, and after following the instructions to execute the  'pressure_diffusion.i' file, I keep getting:

*** ERROR ***
Unable to open file ... Check to make sure that it exists and that you have read permission.

I've tried the other execution methods listed, I've googled around, and I've changed permissions for both the file and the folder it belongs to, but nothing seems to work. I also tried putting it in the simple_diffusion folder from the previous example, but I just received a different error.

Any advice would be greatly appreciated.

**Comment by GiudGiud** [comment, marked as answer]
Hello

Can you please paste the command you are running and the content of the directory with the permissions (ls -l) 

Best,
Guillaume

**Reply by rl3fz** [reply]
command: ../babbler-opt -i pressure_diffusion.i
contents and permissions: pressure_diffusion.i  pressure_diffusion.i.txt

Sorry if the permissions aren't there- I'm using Windows, and the most analogous commands I could find were 'dir' and 'ls', which gave me the same list

**Reply by GiudGiud** [reply]
You mean Windows Subsystem for Linux right?
If you have ls, you could try `ls -l`

**Reply by rl3fz** [reply]
Yes, thank you; it gave me the following:

total 8
---------- 1 rl3fz rl3fz 1497 Jun  3 15:24 pressure_diffusion.i
---------- 1 rl3fz rl3fz 1550 Jun  4 18:17 pressure_diffusion.i.txt

**Reply by GiudGiud** [reply]
-------- means you have no permissions over these files. It should look something like -rw-rw-r-- giving you and your user group the right to write and read those files. 

So could you please use `chmod +rw pressure_diffusion*` then report back 
-the `ls -l` output
-whether you can run the input file now

**Reply by rl3fz** [reply]
ls -l output:
-rw-r--r-- 1 rl3fz rl3fz 1497 Jun  3 15:24 pressure_diffusion.i
-rw-r--r-- 1 rl3fz rl3fz 1550 Jun  4 18:17 pressure_diffusion.i.txt

And it worked! Solve Converged! Thank you very much!

**Comment by GiudGiud** [comment]
Note that a common reason for permissions to be broken in WSL is that you are editing these files, then saving them, from a Windows text editor. I am not sure what the fix is. It's actually working fine on my machine with Atom.
